### ISSUE TITLE
Implement support for Facebook and other identify providers

### ISSUE THREAD

**Issue by jvence** [iainporter/oauth2-provider#15]
Would be great to implement server-side support for authentication through Facebook, Twitter, LinkedIn, etc.

**Comment by jvence** [comment]
Any update on this? Thanks

**Comment by iainporter** [comment]
Sorry, just noticed this.
I have been snowed under with other work.
I  hope to get back this project in May

**Comment by jvence** [comment]
Sorry to press on this. Any update on this?

**Comment by jvence** [comment]
Hi Iain,
Just wanted to follow up to see if you're still supporting this framework and in particular any plans to work on this issue. Thanks

**Comment by iainporter** [comment]
Good grief it's been 8 months since you raised this. 
I guess it's been a low priority as a lot of other work has taken over.
The reality is that I am probably not going to get to it unless some work related activity calls for it.

**Comment by jvence** [comment]
That's too bad. I really think this framework has lots of legs.

**Comment by chartrand22** [comment]
With a littte bit of work, you can merge ian's other fb implementation from his rest project into this one. It requires some rejigging but I did it. 

You can't expect to have it all without some work.

**Comment by iainporter** [comment]
I am happy to work with you guys if you want to generate some pull requests

**Comment by jvence** [comment]
@chartrand22 Would have done it if I knew how. Why not generate a pull request with your fb implementation?

BTW I did submit a pull request for CORS that was accepted and merged into the project so it's not for lack of wanting to do any work

**Comment by jvence** [comment]
@chartrand22 BTW I did submit a pull request for CORS that was accepted and merged into the project so it's not for lack of wanting to do any work

**Comment by chartrand22** [comment]
Yeah I'd have to find the time to do this.  I've implemented quite a bit since then so it's not trivial to "extract" the relevant stuff only.
